src/near-rpc-provider.spec.ts

```diff
@@ -262,6 +262,29 @@ describe('NearRpcProvider', () => {
     })
   })
 
+  describe('getContractState', () => {
```

Comment:
let's add checks to the test verifying that the `state.values` has length > 0, and that `block_height` and `block_hash` are populated with values.